pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java

```diff
@@ -972,4 +1018,31 @@ public void onUpdate(String path, Map<String, String> data, Stat stat) {
         return dynamicConfigurationMap;
     }
 
-}
+    private void processPendingLoadTopic() {
+        lock.lock();
```

Comment:
Why do we need this lock? Would it be enough to `poll()` one item out of the queue and process it?